main.tf

```diff
@@ -14,7 +14,8 @@ locals {
 ##############################################################
 
 module "account_iam_resources" {
-  source = "./modules/account-iam-resources"
+  source = "terraform-redhat/rosa-hcp/rhcs//modules/account-iam-resources"
```

Comment:
This is merged on `main` branch, don't we want to keep there the relative paths ?

Reply:
This generates registry docs which end up pointing to the wrong place

Reply:
I'd rather also keep it local, but idk if it is proper considering the registry

Reply:
`main` branch should keep it local
We should change this to absolute link only when performing a release IMO